language/move-lang/src/typing/core.rs

```diff
@@ -817,6 +822,14 @@ pub fn make_function_type(
                 ])
             }
         }
+        FunctionVisibility::Friend(_) => {
+            // TODO
+            let internal_msg = "Calling a friend function is not implemented yet".to_owned();
```

Comment:
panic for now? `unimplemented!()`

Reply:
I guess the reason I did not make it panic ie because I try to keep the tests passing. But in any case, this will be updated in a PR soon enough.

Reply:
Why bother another PR if it can be added in this PR :) Added in commit 8e3d856.